src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheck.java

```diff
@@ -109,6 +92,64 @@
  * <pre>
  * &lt;module name="UnusedImports"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * import java.awt.Component; // limitation as it match field name in code
```

Comment:
Please put training comment on line above this import.
For import line make training comment to be `// OK`.

Reply:
done